recipes/octo-encryption-cpp/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+
+class OctoEncryptionCPPConan(ConanFile):
+    name = "octo-encryption-cpp"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ofiriluz/octo-encryption-cpp"
+    description = "Octo encryption library"
+    topics = ("cryptography", "cpp")
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
```

Comment:
```suggestion
```
Not needed anymore.

Reply:
ya, i revamped the conanfile post your suggestion, this is already changed :)